sloth.py

```diff
@@ -559,6 +559,14 @@ def generateSet(self, path, setname = None, cutextension = None):
 
 		# retrieve all maps by type
 		for filename in filelist:
+			# check if known picture format
+			try:
+				Image.open(os.path.join(abspath, filename))
+			except OSError:
+				continue
```

Comment:
Perhaps add a debug log saying that you're skipping a file with the contents of the exception?